dev.js

```diff
@@ -22,8 +21,11 @@ const {
     },
   },
 });
-
-const uno = createGenerator({ presets: [presetWarp( { ...options, development: true } )] });
+const asyncPreset = await presetWarp({
+  ...options,
+  development: true,
+});
+const uno = createGenerator({ presets: [asyncPreset] });
```

Comment:
I'm still wondering if this particular script should still include all classes, also those that are part of warp.css (current components.css). We'd probably need to pass those `classesToPurge` to safelist here then. 
It's just that if we don't generate css for classes that are always externalized in this script or in tests, we won't see if our rules still work correctly, and those are used in @warp-ds/css to generate that warp.css stylesheet. Maybe there's a different way to solve it so it's more of a question what you think about it.

Reply:
If `development: true` is set we will generate css for any class you want to test. Or do you mean something else?

Reply:
Ah, of course. Sorry!